skbio/stats/ordination/_principal_component_analysis.py

```diff
@@ -0,0 +1,207 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import numpy as np
+from scipy.linalg import eigh, svd
+from scipy.sparse.linalg import svds
+from ._ordination_results import OrdinationResults
+from skbio.table._tabular import _create_table, _create_table_1d, _ingest_table
+
+
+def pca(
+    X,
+    method="svd",
```

Comment:
I am thinking that the default method should be 'eigh', just to be consistent with `pcoa`, and also because sklearn's PCA defaults to SVD, and our method can be different by default.